transcom/mymove
# B 21077 move shipment button int

## [B-21077](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1010030)

## Summary

This PR moves the "add new shipment" button over to the right for the TOO  to match the Service Counselor's "add new shipment" button for better consistency throughout the app.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Login as TOO
2. Click any existing move
3. Confirm "add new shipment" button is aligned to the right of the page*
*similar to the SC "add new shipment" button on their move details page

### Frontend


## Screenshots
_**(Attachment from agility)  TOO before**_
<img width="838" alt="Screenshot 2024-10-02 at 12 09 19 PM" src="https://github.com/user-attachments/assets/fab7bcc2-948d-4638-8763-d821a3e19717">

_**TOO  after**_
<img width="1054" alt="Screenshot 2024-10-02 at 1 53 47 PM" src="https://github.com/user-attachments/assets/ac611555-1611-44f9-96d8-ee89d9765760">
